pwndbg/gdblib/arch.py

```diff
@@ -19,9 +19,26 @@
     "sparc": "sparc",
     "arm": "arm",
     "armcm": "thumb",
-    "riscv:rv64": "riscv",
 }
 
+# https://github.com/Gallopsled/pwntools/pull/2177
+pwnlib_version = list(map(int, pwnlib.__version__.split(".")[:2]))
+if pwnlib_version[0] == 4 and pwnlib_version[1] < 11:
```

Comment:
We do control the Pwntools version, can we just update it and don't bother about its old implementation?

Reply:
Let's create another PR after this that would update pwntools and remove that branch here.